src/Configuration/PheanstalkConfiguration.php

```diff
@@ -2,23 +2,25 @@
 
 namespace Equip\BeanstalkdConsumer\Configuration;
 
-use Aura\Cli\Context;
 use Auryn\Injector;
-use Pheanstalk\Pheanstalk;
 use Equip\Configuration\ConfigurationInterface;
+use Equip\Env;
+use Pheanstalk\Pheanstalk;
 
 class PheanstalkConfiguration implements ConfigurationInterface
 {
+    const HOST_KEY = 'BEANSTALKD_HOST';
+    const PORT_KEY = 'BEANSTALKD_PORT';
+
     public function apply(Injector $injector)
     {
         $injector->delegate(Pheanstalk::class, [$this, 'getPheanstalk']);
```

Comment:
Any particular reason this is a delegate instead of:

1. Injecting `Env`
2. Using `define` to set the parameters for `Pheanstalk`